tchau/array/array_ex1.cpp

```diff
@@ -0,0 +1,75 @@
+#include <iostream>
+#include <cstring>
+
+using namespace std;
+
+#define MIN 1
+#define MAX 100
+#define DEBUG 0
+
+
+/*
+* Function: findMissedNumberInIntegerArray
+*
+* Question: How do you find the missing number in a given integer array of 1 to 100? 
+*/
+
+
+int findMissedNumberInIntegerArray(int arr[], int min, int max)
+{
+	int sum = 0;
+	int tmpSum = 0;
+	for(int i=min; i<=max; i++)
+	{
+		tmpSum+=i;
+		sum+=arr[i-1];
+#if DEBUG
```

Comment:
Avoid this kind of check in simple programs, functions. A macro function maybe used for printing such debug information.